src/Evolution/Protocols.hpp

```diff
@@ -28,4 +28,19 @@ struct NumericInitialData {
 };
 
 }  // namespace protocols
+
+// FIXME: Is there a way to avoid adding this lazy metafunction?
+template <typename Type, typename = std::void_t<>>
+struct get_import_fields_impl {
```

Comment:
Is it possible to revise your implementation to avoid using a protocol?

Reply:
The protocol is still needed to check if we're using numeric initial data, an analytic solution or analytic data. But I simplified it a lot so the choice _which_ fields are imported from the data file is just made in the Metavariables. Once we need the flexibility to change these fields, we can revisit how to do that.